src/App.js

```diff
@@ -1,83 +1,129 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 const dogStyle = {
-    color: 'brown'
-};
+	color: 'brown'
+}
 
 const catStyle = {
-    color: 'blue'
-};
+	color: 'blue'
+}
 
+// Show user info
 const UserInfo = props => {
-    const user = props.user;
-
-    return (
-        <div>
-            <img src={user.picture.large} />
-            <div>
-                {user.name.first + ' ' + user.name.last}
-            </div>
-        </div>
-    );
-};
+	const user = props.user;
+
+	return (
+		<div>
+			<img src={user.picture.large} />
+
+			<div>
+				{user.name.first + ' ' + user.name.last}
+			</div>
+		</div>
+	)
+}
+
+// List Dog and Cat people
+const UserList = props => {
+	const users = props.users;
+	const removeUser = props.handler;
+
+	return (
+		<div>
+			<h2>
+				{props.list} People
+			</h2>
+
+			<ul>
+				{
+					users.map(function(user, index) {
+						return (
+							<li>
+								<UserInfo user={user} />
+								<a onClick={() => removeUser(index, props.list)}>
```

Comment:
This is a good implementation. No real need to use an `<a>` instead of a `<div>` in this case, though. Probably also best to use something like `listName` or `listType` instead of `list` to imply that the variable is a string.